README.md

```diff
@@ -27,10 +27,21 @@ $ hexo migrate github-issue owner/repo [--publish]
 It will migrate owner/repo issue to hexo, there are some specific issue label is supported:
 
 - "category_", set the sepecfic category for the post
-- "top_", set the top priority for the post.
 - "draft", the post layout will be set to "draft", that means the post will be stored in draft dir
 - "publish", if user migrate with "--publish" opt, only post with "publish" label will be created.
 
```

Comment:
## Front-matter Support

If you specify the front-matter in the issue content, like:
```
---
cover: http://demo.jpeg
top: 1
---
# Title
Hello World!
```
The front-matter will be transparently added in your post writings.

Note that, the front-matter in the issue content has the highest priority, that means the internal front-matter which generated by migrator (such as, `title`, `tags`, `number`, `date`) will also be replaced.